scripts/rook.sh

```diff
@@ -5,14 +5,31 @@ ROOK_DEPLOY_TIMEOUT=${ROOK_DEPLOY_TIMEOUT:-300}
 ROOK_URL="https://raw.githubusercontent.com/rook/rook/${ROOK_VERSION}/cluster/examples/kubernetes/ceph"
 ROOK_BLOCK_POOL_NAME=${ROOK_BLOCK_POOL_NAME:-"newrbdpool"}
 ROOK_CEPH_CLUSTER_VERSION="v14.2.10"
+KUBECTL_RETRY=5
+KUBECTL_RETRY_DELAY=10
 
 rook_version() {
 	echo "${ROOK_VERSION#v}" | cut -d'.' -f"${1}"
 }
 
+kubectl_retry() {
+    local retries=0
+
```

Comment:
In case there is a failure, it may be needed to capture stdout and stderr to prevent it from 'polluting' the output of the function. If this is needed, I do not know yet, it depends if stdout stays 'clean' on a failure.

One idea to do this:
```bash
local stdout stderr
stdout=$(mktemp)
stderr=$(mktemp)

while ! (kubectl "${@}" 2>${stderr} >${stdout}
...
```
And before the `return` do a `cat ${stdout} ; cat ${stderr} >/dev/stderr ; rm -f ${stdout} ${stderr}`